assets/javascripts/test_result.js

```diff
@@ -749,11 +749,37 @@ function renderInvestigationTab(response) {
         keyElement.style.verticalAlign = 'top';
         keyElement.appendChild(document.createTextNode(key));
 
+        var text_lines = response[key].split('\n');
+        var text_lines_rest;
+
+        if (text_lines.length > 10){
+            text_lines_rest = text_lines.slice(10, text_lines.length);
+            text_lines = text_lines.slice(0, 10);
+        }
+
         var valueElement = document.createElement('td');
         var preElement = document.createElement('pre');
-        preElement.appendChild(document.createTextNode(response[key]));
+        preElement.appendChild(document.createTextNode(text_lines.join('\n')));
         valueElement.appendChild(preElement);
 
+        if (text_lines_rest) {
+            var preElementMore = document.createElement('pre');
+            preElementMore.appendChild(document.createTextNode(text_lines_rest.join('\n')));
+            preElementMore.id = 'nextLines_' + key;
+            preElementMore.style = "display: none";
+
+            var moreLink = document.createElement('a');
+            moreLink.style = "cursor:pointer";
+            moreLink.innerHTML = "read more...";
```

Comment:
would you mind using `…` (single unicode character) instead of `...` (three dots)?

Reply:
I would also prefer "Show more". (The JavaScript will not actually read it.)